### PR TITLE
Multi instance behavior complete condition on child execution

### PR DESCRIPTION
**Pull request by tinabandaloemruli**
#### Check List:
* Unit tests:  NO
* Documentation: YES

added new abstract method getCompletionConditionExecution in MultiInstanceActivityBehavior to make it possible to easy override it and enable checking completionConditionSatisified on child execution

### CODE REVIEW

File: modules/flowable-engine/src/test/java/org/flowable/engine/test/cfg/multitenant/SharedExecutorServiceAsyncExecutorTest.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
+package org.flowable.engine.test.cfg.multitenant;
```
Comment (dbmalkovsky) [inline modules/flowable-engine/src/test/java/org/flowable/engine/test/cfg/multitenant/SharedExecutorServiceAsyncExecutorTest.java:1]: Please add the common file license header to any new files; thanks.

File: modules/flowable-engine/src/test/java/org/flowable/engine/test/cfg/multitenant/SharedExecutorServiceAsyncExecutorTest.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+import org.flowable.common.engine.impl.cfg.multitenant.TenantInfoHolder;
```
Comment (dbmalkovsky) [inline modules/flowable-engine/src/test/java/org/flowable/engine/test/cfg/multitenant/SharedExecutorServiceAsyncExecutorTest.java:3]: The project has a established a standard order for imports.   You can find default settings for Eclipse and IntelliJ in the directory `.ide-settings` in the root of the project.  

For example, import ordering is 
* static imports
* java
* javax
* org
* com
* everything else

Please order the imports accordingly; thanks.

File: modules/flowable-engine/src/test/java/org/flowable/engine/test/cfg/multitenant/SharedExecutorServiceAsyncExecutorTest.java (hunk 1 above)
Comment (tinabandaloemruli) [reply modules/flowable-engine/src/test/java/org/flowable/engine/test/cfg/multitenant/SharedExecutorServiceAsyncExecutorTest.java:13]: thanks for the hint I added it

File: modules/flowable-engine/src/test/java/org/flowable/engine/test/cfg/multitenant/SharedExecutorServiceAsyncExecutorTest.java (hunk 2 above)
Comment (tinabandaloemruli) [reply modules/flowable-engine/src/test/java/org/flowable/engine/test/cfg/multitenant/SharedExecutorServiceAsyncExecutorTest.java:15]: imported code style format from flowable guess this is enough

Comment (dbmalkovsky) [reply modules/flowable-engine/src/test/java/org/flowable/engine/test/cfg/multitenant/SharedExecutorServiceAsyncExecutorTest.java:15]: Once you load the code style file to rearrange the imports you need to use `Code`->`Optimize Imports` from the menu or on Linux, `Ctrl+Alt+O`.

**Review by filiphr — CHANGES_REQUESTED** [review]
Thanks for the PR @tinabandaloemruli.

I've left a comment about the current implementation.

Additionally, why are there changes to the  `SharedExecutorServiceAsyncExecutor.java` file? I don't see how this is related to the goal of the PR.

**Pull request closed without merging**